internal-dns/src/bin/dns-server.rs

```diff
@@ -13,23 +13,34 @@
 use anyhow::anyhow;
 use anyhow::Context;
 use clap::Parser;
+use std::net::{SocketAddr, SocketAddrV6};
 use std::path::PathBuf;
 use std::sync::Arc;
 
 #[derive(Parser, Debug)]
 struct Args {
     #[clap(long)]
     config_file: PathBuf,
+
+    #[clap(long)]
+    server_address: SocketAddrV6,
+
+    #[clap(long)]
+    dns_address: SocketAddrV6,
 }
 
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
     let args = Args::parse();
     let config_file = &args.config_file;
+    let dns_address = &args.dns_address;
     let config_file_contents = std::fs::read_to_string(config_file)
         .with_context(|| format!("read config file {:?}", config_file))?;
-    let config: internal_dns::Config = toml::from_str(&config_file_contents)
-        .with_context(|| format!("parse config file {:?}", config_file))?;
+    let mut config: internal_dns::Config =
+        toml::from_str(&config_file_contents)
+            .with_context(|| format!("parse config file {:?}", config_file))?;
+
+    config.dropshot.bind_address = SocketAddr::V6(args.server_address);
     eprintln!("{:?}", config);
```

Comment:
This eprintln is intentional?

Reply:
No big deal, I just wanted to flag it as a possible stray. Seems fine to keep if you'd like.